OSDOCS-16607 [ROSA]: Vale and Dita work for Networking (#110996)
openshift/openshift-docs: modules/running-network-verification-manually-cli.adoc

Reviewer: After the callouts were replaced, the procedure isn't looking good. Turns out, the former callouts should be level **** list items :) Which is probably not even supported by the Customer portal, not sure.

I'll suggest several changes that will pull the whole procedure 1 level up and make it more lightweight.

Reviewer: ```suggestion
* Option 2: Verify the network configuration by specifying the VPC subnets IDs. Replace `<region_name>` with your AWS region and `<AWS_account_ID>` with your AWS account ID:
```

[id="running-network-verification-manually-cli_{context}"]
= Running the network verification manually using the CLI

[role="_abstract"]
You can manually run the network verification checks for an existing {product-title} cluster by using the ROSA CLI (`rosa`).

> When you run the network verification, you can specify a set of Virtual Private Cloud (VPC) subnet IDs or a cluster name.

Reviewer: ```suggestion
To run the network verification, you can specify either a cluster name or a set of Virtual Private Cloud (VPC) subnet IDs.
```
Changing the order because the procedure introduces 2 methods in this order.

.Procedure

> * Verify the network configuration by using one of the following methods:

Reviewer: ```suggestion

```
Can remove this as it's an intro and not a step.

> ** Verify the network configuration by specifying the cluster name. The subnet IDs are automatically detected. Replace `<cluster_name>` with the name of your cluster:

Reviewer: ```suggestion
* Option 1: Verify the network configuration by specifying the cluster name. The subnet IDs are automatically detected. Replace `<cluster_name>` with the name of your cluster:
```
This will become our level 1 step.

I: Run the following command to wait for verification to all subnets to complete:
rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
----
** Ensure that verification to all subnets completes:
+
[source,terminal]
----
$ rosa verify network --watch \
                      --status-only \
                      --region <region_name> \
                      --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
----
+
*** The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state.
*** The `status-only` flag does not trigger a run of network verification but returns the current state, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets.
*** Use the `region` flag to provide a specific AWS region that overrides the `_AWS_REGION_` environment variable.

> *** Use the `subnet-ids` flag to enter a list of subnet IDs separated by commas to verify. If any of the subnets do not exist, the error message `Network verification for subnet 'subnet-<subnet_number> not found` displays and the system does not check subnets.

Reviewer 2: Same question here re: callout replacement

I: Run the following command to wait for verification to all subnets to complete:
rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
----
*** Ensure that verification to all subnets completes:
+
[source,terminal]
----
$ rosa verify network --watch \
                      --status-only \
                      --region <region_name> \
                      --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
----
+

> The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state. The `status-only` flag does not trigger a run of network verification but returns the current state, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets. Use a specific AWS region that overrides the _AWS_REGION_ environment variable with `<region_name>`. Enter a list of subnet IDs separated by commas to verify. If any of the subnets do not exist, the error message `Network verification for subnet 'subnet-<subnet_number> not found` displays and the system does not check subnets.

Reviewer: ```suggestion
** The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state.
** The `status-only` flag does not trigger a run of network verification but returns the current state, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets.
** Use the `region` flag to provide a specific AWS region that overrides the `_AWS_REGION_` environment variable.
** Use the `subnet-ids` flag to enter a list of subnet IDs separated by commas to verify. If any of the subnets do not exist, the error message `Network verification for subnet 'subnet-<subnet_number> not found` displays and the system does not check subnets.
```
As per the OCP callouts guidelines: https://docs.google.com/document/d/1j78SA8Y-ZRlVATbVe6D8bdYj7mQzzhOJtGt1xH_3qa0.

The suggestion can be improved, the list items could be made "more parallel".

I: Run the following command to wait for verification to all subnets to complete:
rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
----

> *** Ensure that verification to all subnets completes:

Reviewer: ```suggestion
** Ensure that verification to all subnets completes:
```

+
[source,terminal]
----
$ rosa verify network --watch \
                      --status-only \
                      --region <region_name> \
                      --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
----
+
** The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state.
** The `status-only` flag does not trigger a run of network verification but returns the current state, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets.
** Use the `region` flag to provide a specific AWS region that overrides the `_AWS_REGION_` environment variable.
** Use the `subnet-ids` flag to enter a list of subnet IDs separated by commas to verify. If any of the subnets do not exist, the error message `Network verification for subnet 'subnet-<subnet_number> not found` displays and the system does not check subnets.

> +

Reviewer: ```suggestion
*** The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state.
*** The `status-only` flag does not trigger a run of network verification but returns the current state, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets.
*** Use the `region` flag to provide a specific AWS region that overrides the `_AWS_REGION_` environment variable.
*** Use the `subnet-ids` flag to enter a list of subnet IDs separated by commas to verify. If any of the subnets do not exist, the error message `Network verification for subnet 'subnet-<subnet_number> not found` displays and the system does not check subnets.
+
```

I: Run the following command to wait for verification to all subnets to complete:
rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
----

> *** Ensure that verification to all subnets completes:

Reviewer: ```suggestion
** Ensure that verification to all subnets completes:
```
